automl/docs/index.rst

```diff
@@ -1,4 +1,4 @@
-.. include:: /../automl/README.rst
+.. include:: README.rst
```

Comment:
Hmm, I was sure I had tried that when I did the work to break up the docs tree.  I'm glad it works both ways!